vyper/ast/grammar.lark

```diff
@@ -167,7 +167,7 @@ pass_stmt: _PASS
 break_stmt: _BREAK
 continue_stmt: _CONTINUE
 
-log_stmt: _LOG NAME "(" [arguments] ")"
+log_stmt: _LOG (NAME | variable_access) "(" [arguments] ")"
```

Comment:
Was this not failing before?

Reply:
no, previously only `log Event()` was allowed